smart_campus/app/models.py

```diff
@@ -227,9 +232,44 @@ def __str__(self):
             category=str(self.category)
         )
 
+    def get_primary_image(self):
+        """Get the primary image
+
+        Returns:
+            image_url (str) if the station primary image exist, None otherwise.
+
+        """
+        primary_image = StationImage.objects.filter(
+            station=self,
+            is_primary=True
+        ).first()
+
+        if primary_image:
+            return primary_image.image.url
+        return None
+
+    def get_other_images(self):
+        """Get the image excluding primary one
+
+        Returns:
+            image_url(:obj: `list` of :obj: `str`)
```

Comment:
I suggest you follow the style in line 239.
This is reStructText format.
I don't think we need to use it here.

Reply:
But how I explain the dictionary(json) column in this way?

Reply:
`dict` itself would be sufficient.
Or if you wanna follow pep484, you could try `Dict[str, any]`

Reply:
Okay.